pkg/handlers/handlers_common.go

```diff
@@ -0,0 +1,18 @@
+package handlers
+
+import (
+	"fmt"
+	"net/http"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func errorMessageResponse(err error, w http.ResponseWriter) {
```

Comment:
I like this as an idea :+1:, we can do a follow up PR refactoring upgrade policies to use this

Reply:
the upgrade policy is also in the handler package, thus I moved this out from the upgrade policy file so we know it is a shared function :D